src/recipe/Baked.Recipe.Service.Application/Database/MySql/MySqlDatabaseFeature.cs

```diff
@@ -15,14 +16,17 @@ public void Configure(LayerConfigurator configurator)
             services.AddSingleton<ITransaction, FlatTransaction>();
         });
 
+        configurator.ConfigureFluentConfiguration(fluent =>
+        {
+            if (_autoUpdateSchema)
+            {
+                fluent.ExposeConfiguration(c => new SchemaUpdate(c).Execute(false, true));
+            }
+        });
+
         configurator.ConfigurePersistence(persistence =>
         {
-            persistence.AutoUpdateSchema = _autoUpdateSchema;
-            persistence.Configurer =
-                MySQLConfiguration.Standard
-                    .ConnectionString(_connectionString)
-                    .Dialect<CustomMySQL57Dialect>()
-                    .MaxFetchDepth(1);
+            persistence.Configurer = MySQLConfiguration.Standard.ConnectionString(_connectionString).Dialect<CustomMySQL57Dialect>();
```

Comment:
line too long, keep this one (not the others) as multiline. dialect makes it long

Reply:
i meant like this;

```csharp
MySQLConfiguration.Standard
    .ConnectionString(_connectionString)
    .Dialect<CustomMySQL57Dialect>();
```